### DISCUSSION TITLE
What are experimental globe features?

### DISCUSSION THREAD

**Discussion by KjartanOli** [GitSquared/edex-ui#968] [Questions]
This option catches my eye every time I open up the settings menu. But I've tried turning it on and never noticed any changes. What are they supposed to do?

**Comment by GitSquared** [comment]
Good question... If I remember right there used to be some kind of visual effects where a line was drawn between points to represent active upload/download states. Think it might have been scrapped since then, i'd have to dig into the code to check whether that setting really affects anything still.

**Comment by Termuxkkk** [comment]
no
